### PR TITLE
NMS-13063: applicationChanged/Created events are missing

### PR DESCRIPTION
**Pull request by mfuhrmann**
### All Contributors

* [x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [x] Have you made a comment in that issue which points back to this PR?
* [x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-13063

### CODE REVIEW

**Comment by mfuhrmann** [comment]
@indigo423 related to https://github.com/OpenNMS/opennms/pull/3314#discussion_r597540071 . Does https://github.com/OpenNMS/opennms/pull/3373/files#diff-e5078b4f9f6297992d27fed10b3cdc7c75f10921c8b077fb5bfc8084ce0311aeR261 sounds better?

**Review by christianpape — APPROVED** [review]
LGTM.

**Review by indigo423 — APPROVED** [review]
LGTM! 👍

File: opennms-base-assembly/src/main/filtered/etc/events/opennms.internal.events.xml [hunk 1]
```diff
@@ -248,10 +248,25 @@
...
+      <logmsg dest="logndisplay">Application '%parm[applicationName]%' has been deleted.</logmsg>
       <severity>Warning</severity>
    </event>
+   <event>
+      <uei>uei.opennms.org/internal/applicationChanged</uei>
+      <event-label>OpenNMS-defined node event: applicationChanged</event-label>
+      <descr>The application '%parm[applicationName]%' with ID '%parm[applicationId]%' has been changed.</descr>
+      <logmsg dest="logndisplay">Application '%parm[applicationName]%' configuration has been changed.</logmsg>
+      <severity>Warning</severity>
+      <operinstruct>Make sure '%parm[applicationName]%' application's definition still reflects the requirements. Please verify it's &lt;a href=&quot;admin/applications.htm?applicationid=%parm[applicationId]%&amp;edit=services&quot;>definition&lt;/a>.</operinstruct>
```
[the remark this answers is not in the document]
Comment (mfuhrmann) [reply opennms-base-assembly/src/main/filtered/etc/events/opennms.internal.events.xml:261]: Using the `>` in this XML file definitely works. I always was confused why it worked. But let's do like it should look like.

**Comment by RangerRick** [comment]
Is there anything holding up merging this for Horizon 28, or should we go for it?

**Pull request merged**